plugins/source/snyk/client/client.go

```diff
@@ -0,0 +1,51 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/pavel-snyk/snyk-sdk-go/snyk"
+	"github.com/rs/zerolog"
+)
+
+type Client struct {
+	*snyk.Client
+
+	OrganizationID string
+	organizations  []string
+
+	logger zerolog.Logger
+}
+
+var _ schema.ClientMeta = (*Client)(nil)
+
+func (c *Client) ID() string {
+	return "snyk/" + c.OrganizationID
```

Comment:
```suggestion
	return c.OrganizationID
```
Unless OrgID can be empty...

Reply:
This is also tied to the spec.Name sometimes in some plugins.

Reply:
orgID shouldn't be empty (only initially?), so will change